scheduled_lambdas.go

```diff
@@ -0,0 +1,166 @@
+package rockset
+
+import (
+	"context"
+	"net/http"
+
+	"github.com/rs/zerolog"
+
+	rockerr "github.com/rockset/rockset-go-client/errors"
+	"github.com/rockset/rockset-go-client/openapi"
+	"github.com/rockset/rockset-go-client/option"
+)
+
+// CreateScheduledLambda creates a new scheduled lambda, with an optional description.
+//
+// REST API documentation https://docs.rockset.com/documentation/reference/createscheduledlambda
+func (rc *RockClient) CreateScheduledLambda(ctx context.Context, workspace, apikey string, cronString string, qlName string,
+	options ...option.ScheduledLambdaOption) (openapi.ScheduledLambda, error) {
+	var err error
+	var httpResp *http.Response
+	var resp *openapi.ScheduledLambdaResponse
+
+	q := rc.ScheduledLambdasApi.CreateScheduledLambda(ctx, workspace)
+	req := openapi.NewCreateScheduledLambdaRequest(apikey, cronString, qlName)
+
+	opts := option.ScheduledLambdaOptions{}
+	for _, o := range options {
+		o(&opts)
+	}
+
+	if opts.Tag != nil {
+		req.Tag = opts.Tag
+	}
+	if opts.TotalTimesToExecute != nil {
+		req.TotalTimesToExecute = opts.TotalTimesToExecute
+	}
+	if opts.Version != nil {
+		req.Version= opts.Version
+	}
+	if opts.WebhookAuthHeader != nil {
+		req.WebhookAuthHeader = opts.WebhookAuthHeader
+	}
+	if opts.WebhookPayload != nil {
+		req.WebhookPayload = opts.WebhookPayload
+	}
+	if opts.WebhookUrl != nil {
+		req.WebhookUrl = opts.WebhookUrl
+	}
+
+	err = rc.Retry(ctx, func() error {
+		resp, httpResp, err = q.Body(*req).Execute()
+
+		return rockerr.NewWithStatusCode(err, httpResp)
+	})
+
+	if err != nil {
+		return openapi.ScheduledLambda{}, err
+	}
+
+	log := zerolog.Ctx(ctx)
+	log.Trace().Str("status", httpResp.Status).Msg("scheduled lambda created")
+
+	return resp.GetData(), nil
+}
+
+// UpdateScheduledLambda updates an existing scheduled lambda, with an optional description.
+//
+// REST API documentation https://docs.rockset.com/documentation/reference/updatescheduledlambda
+func (rc *RockClient) UpdateScheduledLambda(ctx context.Context, workspace string, scheduledLambdaRrn string,
```

Comment:
abbreviations should be fully capitalized
```suggestion
func (rc *RockClient) UpdateScheduledLambda(ctx context.Context, workspace string, scheduledLambdaRRN string,
```